### PR TITLE
TT-1618: Document how to use test tool

### PR DESCRIPTION
**Pull request by J-Lambo**
DO NOT MERGE - questions still to be answered. 

First draft of instructions for the Matching Service Test Tool. 

Author: @J-Lambo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,73 @@
...
+
+Replace the `matchUrl` and `accountCreationUrl` URLs with the same URLs you created for your local matching service. Refer to the [guidance on matching requests for more information about matching and account creation URLs](http://alphagov.github.io/rp-onboarding-tech-docs/pages/ms/msBuild.html#respond-to-json-matching-requests).
+
+## Run
+
+Run `bin/verify-matching-service-test-tool` to start the test tool.
+
+This command will run a series of test scenarios. Successful results will return in green text. Failed results will return in red text with error messages.
+
+:question: Is this necessary to document?
```
Comment (richardTowers) [inline README.md]: Probably not - could just say "will run a series of test scenarios" and segue in to the next section.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,73 @@
...
+
+## Test scenarios
+
+:question: Could we add a list of default scenarios and what they test for?
+
+| Scenario | Description |
+| -------- | ----------- |
+|          |             |
```
Comment (richardTowers) [inline README.md]: Looking at the output, we currently have these "scenarios" - they're not even unique at the moment, so I think they need a bit more descriptive names...

| Scenario | Description |
| -------- | ----------- |
| runForSimpleCase | |
| runForExtensiveCase | |
| runForWhenAllElementsAreVerifiedAndNoMultipleValues | |
| runForExtensiveCase | |
| runCase | |
| runForUserAccountCreation | |

File: README.md [hunk 3]
```diff
@@ -1,16 +1,73 @@
...
+For example, to match the identity of a user with the first name `Sam`, you could amend the `example-match.json` file to include:
+
+```
+},
+"firstName": {
+  "value": "Sam",
+  "verified": true
+```
+
+:question: Is this a clear enough example?
```
Comment (richardTowers) [inline README.md]: Yes, I think so.

File: README.md (hunk 2 above)
Comment (richardTowers) [reply README.md]: Added a separate PR to sort the descriptions of the scenarios out - #7

Comment (richardTowers) [reply README.md]: | Scenario | Description |
| ----- | ----- |
|  Simple request with level of assurance 1 | |
|  Complex request with level of assurance 1 | |
|  Simple request with level of assurance 2 | |
|  Complex request with level of assurance 2 | |
|  Simple request with address missing optional fields | |
|  Simple user account creation request | |

**Comment by J-Lambo** [comment]
@richardTowers Thanks for the speedy review. I've made a few changes based on your comments and added the list of test scenarios.

**Pull request merged**